src/Microsoft.CSharp/src/Microsoft.CSharp.csproj

```diff
@@ -7,6 +7,7 @@
     <RootNamespace>Microsoft.CSharp</RootNamespace>
     <!-- dotnet/corefx#3128 tracks removing this exclusion -->
     <ExcludeLocalizationImport Condition="'$(TargetsWindows)' != 'true'">true</ExcludeLocalizationImport>
+    <LangVersion>latest</LangVersion>
```

Comment:
I believe the langver is common to the whole CoreFx repo. It make sense since the repo also controls the toolset compiler. You do not want to start using some new language feature only to find that it will not build in CI.

I think this change should be reverted. The rest looks good.